tube_tipper.scad

```diff
@@ -22,11 +22,11 @@ module tube_tipper(number_tubes = 5) {
   tube_diameter = 18.2; // diameter of the tubes
   tube_z_stretch = 1; // how much to stretch in vertical direction to account for printing collapse
```

Comment:
looks to me like the tube z stretch works just fine. Easiest way to see if something works in the rendering is to exaggerate it greatly. Try `tube_z_stretch = 1;` and see what happens

Reply:
Yup, exaggeration does make it seem like this is working, I'll kick it up a touch since it seems to not be on strong enough.